.mergify.yml

```diff
@@ -0,0 +1,24 @@
+pull_request_rules:
+  - name: Ask PR author what redirects are needed
+    conditions:
+      - label="redirects-needed"
+    actions:
+      comment:
+        message: |
+          @{{author}}, since a file was deleted from the `src/pages` and/or `src/fragments` directories, redirects will need to be set up so these previous pages do not 404. Please answer these questions:
+
+          - What is the source address of the redirect? (Where are you trying to redirect from?)
+
+          - What is the target address of the redirect? (Where are you trying to redirect to?)
+
+          - Type of redirect? 301 - permanent redirect or 302 - temporary redirect? (More info on Amplify Hosting redirects here: https://docs.aws.amazon.com/amplify/latest/userguide/redirects.html)
```

Comment:
This is the message that the Mergify bot will add to the PR as a comment. Does anyone have any thoughts/feedback on this messaging?